client/src/components/CitizenRequest/CitizenRequest.component.tsx

```diff
@@ -4,14 +4,34 @@ import { Agency } from '../../services/AgencyService'
 import { Enquiry, Mail, postMail } from '../../services/MailService'
 import { EnquiryModal } from '../EnquiryModal/EnquiryModal.component'
 import { useStyledToast } from '../StyledToast/StyledToast'
+import { useGoogleAnalytics } from '../../contexts/googleAnalytics'
+import * as FullStory from '@fullstory/browser'
 
 const CitizenRequest = ({ agency }: { agency: Agency }): JSX.Element => {
   const toast = useStyledToast()
   const {
-    onOpen: onDeleteModalOpen,
-    onClose: onDeleteModalClose,
-    isOpen: isDeleteModalOpen,
+    onOpen: onEnquiryModalOpen,
+    onClose: onEnquiryModalClose,
+    isOpen: isEnquiryModalOpen,
   } = useDisclosure()
+  const googleAnalytics = useGoogleAnalytics()
+
+  const sendOpenEnquiryEventToAnalytics = () => {
+    googleAnalytics.sendUserEvent(
+      googleAnalytics.GA_USER_EVENTS.OPEN_ENQUIRY,
+      agency.email,
+    )
+    FullStory.event(googleAnalytics.GA_USER_EVENTS.OPEN_ENQUIRY, {
+      // property name uses `ident_type` pattern
```

Comment:
```suggestion
```

Reply:
Comment addressed in commit https://github.com/opengovsg/askgovsg/pull/249/commits/0abdfd99aa6fb009a6b871b4059df30c6d876026.